mappings/net/minecraft/client/toast/SystemToast.mapping

```diff
@@ -22,4 +33,21 @@ CLASS net/minecraft/class_370 net/minecraft/client/toast/SystemToast
 	METHOD method_27025 addWorldDeleteFailureToast (Lnet/minecraft/class_310;Ljava/lang/String;)V
 		ARG 0 client
 		ARG 1 worldName
+	METHOD method_29046 drawPart (Lnet/minecraft/class_4587;Lnet/minecraft/class_374;IIII)V
+		ARG 1 matrices
+		ARG 2 manager
+		ARG 3 width
+		ARG 4 textureV
+		ARG 5 y
+		ARG 6 height
+	METHOD method_29047 newSystemToast (Lnet/minecraft/class_310;Lnet/minecraft/class_370$class_371;Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;)Lnet/minecraft/class_370;
```

Comment:
`create`? `copy`?

Reply:
more `create` than `copy`, it just ensures that while giving you a new instance of SystemToast it has the right width.

Reply:
is this an instance method or a static method? Should rename to `create` if this is just a factory static method.

Reply:
It's a static method

Reply:
```suggestion
	METHOD method_29047 create (Lnet/minecraft/class_310;Lnet/minecraft/class_370$class_371;Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;)Lnet/minecraft/class_370;
```